enterprise/admin/forms.py

```diff
@@ -31,6 +33,10 @@
     EnterpriseCustomerReportingConfiguration,
 )
 from enterprise.utils import MultipleProgramMatchError, decrypt_string
+try:
+    from third_party_auth.models import SAMLProviderConfig as saml_provider_configuration
+except ImportError:
```

Comment:
We dont need this try-catch for edx-platform imports since enterprise is part of edx-platform.

Reply:
Agree but I think, Test suits are not aware of that.